### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by DariaVeretyak**
https://dariaveretyak.github.io/js_2048_game/

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,347 @@
...
+  } else {
+    newValue = 2;
+  }
+}
+
+let row;
+let column;
+// проверяем на пустоту
+const selectEmptyCell = () => {
+  row = getRandomCell(0, field.length);
```
Comment (ivanchuk-yurii) [inline src/scripts/main.js:124]: `row = getRandomCell`, if this function is used not only for cells, rename it

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,347 @@
...
+      };
+    }
+  }
+
+  return randomValue;
+};
+
+let newValue;
+
+function getRandomValue() {
```
Comment (ivanchuk-yurii) [inline src/scripts/main.js:110]: return value from this function, it's hard to track variable

File: src/scripts/main.js [hunk 3]
```diff
@@ -1,3 +1,347 @@
...
+    newValue = 4;
+  } else {
+    newValue = 2;
+  }
+}
+
+let row;
+let column;
+// проверяем на пустоту
+const selectEmptyCell = () => {
```
Comment (ivanchuk-yurii) [inline src/scripts/main.js:123]: return value from this function, it's hard to track variables

File: src/scripts/main.js [hunk 4]
```diff
@@ -1,3 +1,347 @@
...
+};
+
+// нажатие start
+start.addEventListener('click', (ev) => {
+  addNewValue();
+  addNewValue();
+  getValueField();
+  getFilledCells();
+
+  if (filledCellsArr.length >= 1) {
```
Comment (ivanchuk-yurii) [inline src/scripts/main.js:188]: reason of this `if`?

File: src/scripts/main.js [hunk 5]
```diff
@@ -1,3 +1,347 @@
...
+      if (field[i][j] !== '') {
+        cellField = field[i][j];
+        tRows[i].cells[j].classList.add(`field-cell--${cellField}`);
+      }
+    }
+  }
+};
+
+// обнуляем field
+const getEmptyField = () => {
```
Comment (ivanchuk-yurii) [inline src/scripts/main.js:70]: why this is called `get`, it should be `resetField`

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]
pay attention to variable naming. try to not use global variables when they are needed only in one function. instead use them only in this place as returned value from another function. when I click any button except arrows, numbers are still adding. condition `The merged cell can’t be merged twice during one move` doesn't work if numbers are next to each other. a few bugs when trying to merge numbers on the beginning. try to test in different scenarios 

https://user-images.githubusercontent.com/53611910/191805866-b3952b98-8dda-49ad-a1f9-071d942dfdb1.mov
https://user-images.githubusercontent.com/53611910/191804317-e781d73e-b31f-4694-855c-0d9e07cc2710.mov
https://user-images.githubusercontent.com/53611910/191802706-edf22d95-3890-42ae-b424-08ecbeeb440a.mov

**Review by marina-tilniak — APPROVED** [review]

**Pull request closed without merging**